test/test_tensor_creation_ops.py

```diff
@@ -1199,7 +1199,7 @@ def seed(generator):
             self.assertTrue((res1 < 6).all().item())
             self.assertTrue((res1 >= 0).all().item())
 
-    @dtypes(torch.half, torch.float, torch.double,
+    @dtypes(torch.half, torch.float, torch.bfloat16, torch.double,
```

Comment:
should we also add `@dyptes(torch.bfloat16)` to tests in `test_linalg.py` such as `test_norm_*`?

Reply:
Good idea, let me do it in followup PR